velox/common/encode/Base64.cpp

```diff
@@ -22,6 +22,12 @@
 
 namespace facebook::velox::encoding {
 
+// Size of the binary block before encoding.
+constexpr static int kBinaryBlockSize = 3;
+
+// Size of the encoded block after encoding.
+constexpr static int kEncodedBlockSize = 4;
+
```

Comment:
These names are confusing. I assume these constants represent that every 6 bits of input are encoded using 1 byte of output, i.e. 3 input bytes are encoded using 4 bytes. Is this the case? Maybe update the comments to make this a bit clearer.